### DISCUSSION TITLE
Bug using await ctx.QuerySelectorAllAsync

### DISCUSSION THREAD

**Discussion by tomasml** [ChromiumDotNet/WebView2.DevTools.Dom#15] [General]
If you use this
var links= (dynamic)null;
links = await ctx.QuerySelectorAllAsync<WebView2.DevTools.Dom.HtmlAnchorElement>("MOTHER");
in this line the system stop, no errors, no message, nothing, the code does not continue, just stop in this line.

Also, include a event where the page is really completed loaded.

I want to congratulate you for this good job. Go on.

## Looking for some help? Get priority support by [sponsoring the project](https://github.com/sponsors/amaitland)/

**Before you file a bug, have you:**

* Considered if this is a question and not a bug?. For questions please use Discussions, if unuse use Discussions (https://github.com/ChromiumDotNet/WebView2.DevTools.Dom/discussions).
* Tried upgrading to newest version of WebView2.DevTools.Dom, to see if your issue has already been resolved and released?
* Checked existing open *and* closed issues, to see if the issue has already been reported?

* Tried reproducing your problem in a new isolated project?

### Description

[Description of the issue]

### Complete minimal example reproducing the issue

Complete means the code snippet can be copied into a unit test method in a fresh C# project and run.
Minimal means it is stripped from code not related to reproducing the issue.

E.g.

```csharp
// Arrange
await using var devtoolsContext = await coreWebView2.CreateDevToolsContextAsync();

// Act
...

// Assert
...
```

### Expected behavior:

[What you expect to happen]

### Actual behavior:

[What actually happens]

### Versions

* Which version of WebView2.DevTools.Dom are you using?
* Which .NET runtime and version are you targeting? E.g. .NET framework 4.6.2 or .NET Core 3.1.

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by amaitland** [comment]
Converted to discussion as it's not clear this is actually a bug in `WebView2.DevTools.Dom`, a new issue can be created if required.

Please edit the discussion and complete the bug report. Alternatively submit a `PR` that includes a new `xUnit Test Case` that reliably reproduces the problem.

> in this line the system stop, no errors, no message, nothing, the code does not continue, just stop in this line.

Check the `Windows Event Log (Event Viewer)`, there should be more details of the exception.

> Also, include a event where the page is really completed loaded.

I'll defer to the `WebView2` team to add that functionality to the underlying browser.
